src/endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import * as https from 'https';
```

Comment:
I really don't think making this a browser dependency is a good idea, I mentioned before that the adapter pattern works rather well.